build/toolchain/linux/BUILD.gn

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+
+import("${build_root}/toolchain/gcc_toolchain.gni")
+
+gcc_toolchain("linux_x64_gcc") {
+  toolchain_args = {
+    current_os = "linux"
+    current_cpu = "x64"
+    is_clang = false
+  }
+}
+
+gcc_toolchain("linux_x64_clang") {
```

Comment:
gcc_toolchain for a clang toolchain ? this is misleading...  something like "set_toolchain_args" would be more appropriate unless i don't understand well

Reply:
Really what it means is "gcc or compatible" since clang is a drop in replacement for gcc (contrast with Microsoft's compiler which has totally different arguments).